.github/workflows/remove-long-timeout-labels.yml

```diff
@@ -0,0 +1,32 @@
+name: Remove `CI-long-timeout` labels
+
+on:
+  workflow_run:
+    workflows:
+      - CI
+    types:
+      - completed
+
+env:
+  GH_REPO: ${{ github.repository }}
+  GH_NO_UPDATE_NOTIFIER: 1
+  GH_PROMPT_DISABLED: 1
+
+jobs:
+  remove-labels:
+    runs-on: ubuntu-latest
+    if: >
+      github.repository_owner == 'Homebrew' &&
+      github.event.workflow_run.event == 'pull_request'
+    strategy:
+      matrix:
+        PR: ${{ fromJson(toJson(github.event.workflow_run.pull_requests.*.number)) }}
```

Comment:
Seems to be an actionlint bug?

https://rhysd.github.io/actionlint/#eJyNUstOwzAQvOcrVqVqJSRH4hqpSKUNbcSjUQjnNIFNa0js4EcDqvrvOHEfKXDAJ++MvTs7uywt0YMIS75BWE4CUnC2IoqWyLVaQpFmWEjH4cxzAGou3vOC14nQbXxCpA0BCEwCx0G2aYDZPIn8cOFBf7uFFVVrnbkCKy6p4uILdjv75nGRPIfTceybWxzcBn7kwZWlwmjxEMbJNHga39z70wZ33njWlhOtaGIl2vpGlyRGK+hMM6UNp1CqlqK5B9d7kb+kJLxmKGA0guGcl5gJrIcwGJw/xw0y5XY9sFD7rdJFkQj80KbesP0nlTDVV18HZ8pUCfp5iADC6MyXP7J3c0r30mW6zIzK1rbm5CktSJ5K5ZlrIbGFKxQllZJydpxJk4cc8nhQC6oQLiDnAparNVQC8JWq5V41Vp1hsn+sx7Gj/dQPx4wvXtz5j7ZNiS8CTRuzIJ4/31jm1ErXEGuUG0Zdutk4OImFXj+MekBIdwngh75vRa/gjQ==

Reply:
Or, a reduced example: https://rhysd.github.io/actionlint/#eJxNjsEOgjAQRO98xR5MSEzoB/TqyRvhBwg1K6ClxXZXNIR/t7QQOb6Z3ZmxRmYAk3XPu7ZT7TjyX/EJAQq4XLOHVVEwPCh0mxd+fGGNBFZsiAvdEHqKlicXoP3uIUNDrv/sBFBWEk7zDG1PHSuBbzQkjmPEyFrXDl8cIr04i9QMy7Ll43hYuK4HvHUW8jU1tYmyCuf5D5T9SFM=

Reply:
Will file an issue for this.

Reply:
rhysd/actionlint#294